### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by msamesch**
Maybe there's more..

### CODE REVIEW

File: appendices/migration83/other-changes.xml [hunk 1]
```diff
@@ -48,9 +48,10 @@
    </para>
 
    <para>
-    Memory stream no longer fails if seek offset is past the end. Instead
-    the memory is increase on the next write and date between the old end and
-    offset is filled with zero bytes in the same way how it works for files.
+    A memory stream no longer fails if seek offset is past the end. Instead
+    the memory will be inceased on the next write and the data between the old
+    end and the offset is filled with zero bytes in the same way as it works
+    for files.
```
Comment (Girgias) [inline appendices/migration83/other-changes.xml:54]: Is this possibly even clearer?
```suggestion
    A memory stream no longer fails if the seek offset is past the end.
    Instead, the memory will be inceased on the next write and the data between
    the old end and the offset is filled with zero bytes, similar to how files work.
```

**Comment by msamesch** [comment]
Thanks a lot for reviewing.

File: appendices/migration83/other-changes.xml [hunk 2]
```diff
@@ -48,10 +48,9 @@
    </para>
 
    <para>
-    A memory stream no longer fails if seek offset is past the end. Instead
-    the memory will be inceased on the next write and the data between the old
-    end and the offset is filled with zero bytes in the same way as it works
-    for files.
+    A memory stream no longer fails if the seek offset is past the end.
+    Instead, the memory will be inceased on the next write and the data between
```
Comment (nielsdos) [inline appendices/migration83/other-changes.xml:52]: Still a typo left: inceased instead of increased

Comment (msamesch) [reply appendices/migration83/other-changes.xml:52]: Thanks. Adding typos while fixing others...

**Pull request merged**